conformance/utils/suite/suite.go

```diff
@@ -519,6 +519,12 @@ func (suite *ConformanceTestSuite) Report() (*confv1.ConformanceReport, error) {
 	}
 	defer suite.lock.RUnlock()
 
+	if suite.supportedFeaturesSource == supportedFeaturesSourceManual &&
+		!hasMeshFeatures(suite.SupportedFeatures) &&
+		!suite.conformanceProfiles.HasAny(MeshHTTPConformanceProfileName, MeshGRPCConformanceProfileName) {
+		return nil, fmt.Errorf("can't generate report: Gateway's supported features should be read from Status and not supplied through flags")
```

Comment:
This means its impossible to run the report if your gateway doesn't support the SupportFeatures feature. Which is a brand new feature. This seems far to early to do, if ever